channels.md

```diff
@@ -795,3 +795,18 @@ _Note that the _`Code`_ line is not a continuous sequence of numbers, there are
 | `851` | Vacancselect |
 | `852` | Allyourz |
 | `853` | Booking Expert BeGain |
+| `854` | ANA Sky Holiday |
+| `856` | JAL |
```

Comment:
missing 855

Reply:
@patrikmiksovsky95 yeah I missed that in GA, so there is no 855 right now